tests/functional/sync_v3_sign_up.js

```diff
@@ -70,12 +70,12 @@ define([
         .then(openVerificationLinkInDifferentBrowser(email, 0))
 
         // about:accounts takes over, so no screen transition
-        .then(noPageTransition(selectors.CHOOSE_WHAT_TO_SYNC.HEADER))
+        .then(noPageTransition(selectors.CHOOSE_WHAT_TO_SYNC.HEADER, 5000))
```

Comment:
Little known to me, this was in error before. The page was transitioning, but we weren't waiting long enough to catch it. The transition is fixed by https://github.com/mozilla/fxa-content-server/pull/5546/files#diff-a1ed3eb721932d97da56f4a5117ed165R28